server/router.js

```diff
@@ -95,4 +90,11 @@ function getDefaultAppState () {
   return state
 }
 
+function sendSPA (route, res, state) {
+  const frozenState = Object.freeze(state)
+  const contents = app.toString('/', frozenState)
```

Comment:
should this be `toString(route, ...)` now?  Ideally /:archiveKey should render the "server-rendered properties" page, and have the "archive key" (which is derived from route) in both server and client